packages/server/test/security/api.test.ts

```diff
@@ -5,7 +5,7 @@ import { v4 as uuid } from 'uuid'
 import froth from './mocha-froth'
 
 const UUID_LENGTH = uuid().length
-const TOKEN_LENGTH = 88
+const TOKEN_LENGTH = 125
```

Comment:
We should maybe consider exporting this value where it is defined in the sync schema instead.

Reply:
I'll actually remove this limitation. With the legacy handling tokens can be of more than one size.